dream.js

```diff
@@ -287,7 +287,12 @@ function Dream() {
     if (isValidSchema(schema)) return schema;
 
     if (typeof (schema) === 'string') {
-      var foundSchema = _.findWhere(_schemas, { name: schema });
+      var foundSchema = null;
```

Comment:
Is there a way to do this in a manner that is compatible with both mentioned versions?